src/MSALWrapper.Test/AuthFlow/AuthFlowExecutorTest.cs

```diff
@@ -86,7 +86,7 @@ public void ConstructorWith_Null_AuthFlows()
         public void ConstructorWith_Valid_Arguments()
         {
             var logger = this.serviceProvider.GetService<ILogger<AuthFlowExecutor>>();
-            Action authFlowExecutor = () => new AuthFlowExecutor(logger, this.authFlows);
+            Action authFlowExecutor = () => new AuthFlowExecutor(logger, null, this.authFlows);

```

Comment:
Shouldn't this be non-null for valid args?

Reply:
Yup, that's true. I will change it.